bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -308,14 +308,13 @@
 
 	<!-- Litre Channel -->
 	<channel-type id="amrLitre">
-		<item-type>Number:Litre</item-type>
+		<item-type>Number:Volume</item-type>
 		<label>Litre per second</label>
 		<state pattern="%d %unit%" readOnly="true" />
 	</channel-type>
 
-	<!-- Litre Channel -->
 	<channel-type id="amrCubicMetre">
-		<item-type>Number:Litre</item-type>
+		<item-type>Number:Volume</item-type>
 		<label>Cubic metre</label>
```

Comment:
Shouldn't this now be "Volume" here as the unit can be chosen by the user?

Reply:
Sorry I do not understand your comment. I already changed it to "Volume"

Reply:
My comment was about the label, not the item-type.

Reply:
I changed the label